src/extension/features/accounts/hide-memos/settings.js

```diff
@@ -1,8 +1,9 @@
 module.exports = {
   name: 'HideMemos',
   type: 'checkbox',
-  default: false,
+  default: true,
```

Comment:
Is this temporary, or is the intent to hide memos by default?

Reply:
This is just enabling the feature, we should probably just change the name of the feature in code too, this feature now "Adds the menu as an option of columns to hide in the View menu"